spinner-lib/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
     package="com.duchuyctlk.widget">
 
     <application
-        android:allowBackup="false"
+        android:allowBackup="true"
```

Comment:
So please keep this line unchanged and make it allows Backup in your application only.
I will merge this fix soon. Thanks for your contribution.